django_project/ford3/models/campus.py

```diff
@@ -5,6 +5,11 @@
 from ford3.models.campus_event import CampusEvent
 
 
+
+class ActiveCampusManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted=False)
```

Comment:
@pierrealixt i'm wondering could we filter provider here too?
to make sure only the active provider is selected.
an advanced user may try to exploit deleted provider from campus.

i'm thinking something like:
`return super().get_queryset().filter(provider_deleted=False, deleted=False)`

Reply:
or when we soft delete a provider, we also soft delete its campus.

Reply:
yes, i think it works too.
we will need to soft-delete related campuses and qualifications.
let me add it into #398

Reply:
@pierrealixt i just realized that from the perspective of #398, it doesn't know this PR yet.
so i created #433.
let's get this PR and #398 merged first, then i can do #433